src/services/memcached/ascii.rs

```diff
@@ -0,0 +1,278 @@
+// Copyright 2017 vavrusa <marek@vavrusa.com>
+//
+// Licensed under the MIT License (see MIT-ascii.txt);
+//
+// Copyright 2022 Datafuse Labs
```

Comment:
Please do not change the license header unless you made some big change.
BTW,  I don't think you can add the copyright of Datafuse Labs even you made the change, as this code is hosted in ASF. Please remove the below  license headers.